patches/kaggle_gcp.py

```diff
@@ -183,11 +186,28 @@ def patched_init(self, *args, **kwargs):
         if specified_credentials is None:
             Log.info("No credentials specified, using KaggleKernelCredentials.")
             kwargs['credentials'] = kaggle_kernel_credentials
+        
+        # TODO(vimota): Remove the exclusion of TablesClient once
+        # the client has fixed the error:
+        # `multiple values for keyword argument 'client_info'``
```

Comment:
Is that an actual error that you see if this is set? I was hoping that Python would just ignore the kwargs value if a named value was already passed.

Reply:
Yeah unfortunately it's a type error :( https://stackoverflow.com/questions/21764770/typeerror-got-multiple-values-for-argument